dev_requirements.in

```diff
@@ -1,9 +1,9 @@
 -c requirements.txt
```

Comment:
Is this line supposed to be here? Not your doing, but just looked weird.

Reply:
This looks like it is `pip-compile` specific and used to constrain the `dev_requirements.in` to packages used for "production" (as they call it in the example) in `requirements.txt`, so they are compatible, yet different.
There is an example towards the end under: "Workflow for layered requirements"
https://pypi.org/project/pip-tools/